docs/usage/customization-guide.md

```diff
@@ -976,6 +976,8 @@ The following features are available for matching:
 |                  |              | **`<sysfs-attribute>`** | string | Sysfs network interface attribute, available attributes: `dax`, `rotational`, `nr_zones`, `zoned` |
 | **`system.osrelease`** | attribute |       |            | System identification data from `/etc/os-release` |
 |                  |              | **`<parameter>`** | string | One parameter from `/etc/os-release` |
+| **`system.vendor`** | attribute |       |            | System vendor identification data from `/sys/devices/virtual/dmi/id/` |
+|                  |              | **`name`** | string | Vendor name from `/sys/devices/virtual/dmi/id/sys_vendor` |
```

Comment:
We could just expose the "attributes" as is, not apply any name mapping. Like, say, `storage.block` above. This would then become just `sys_vendor`. This would also prevent any possibility of name clashes in the future. WDYT?

Reply:
>This would then become just sys_vendor. This would also prevent any possibility of name clashes in the future. WDYT?

This makes sense, thanks! Updated so.